common/latch_test.cpp

```diff
@@ -87,7 +87,7 @@ TEST(LatchTest, MultiThreaded) {
     called = true;
   });
 
-  std::deque<std::jthread> threads;
+  std::deque<std::thread> threads;
```

Comment:
```suggestion
  std::vector<std::thread> threads;
```
Is there a specific reason for deque? It just kind of says "look at me" since it's not the default choice.